core/renderers/common/block_rendering.js

```diff
@@ -27,7 +28,16 @@ goog.requireType('Blockly.Theme');
  * @type {boolean}
  * @package
  */
-Blockly.blockRendering.useDebugger = false;
+let useDebugger = false;
```

Comment:
I think since you changed the usages of this the `@package` annotation can be removed.